src/core/streams/stream_message.ts

```diff
@@ -37,4 +38,32 @@ export class StreamMessage {
   get templateChildren() {
     return Array.from(this.templateElement.content.children)
   }
+
+  activateScriptElements(parentElement: StreamElement) {
+    for (const inertScriptElement of ((parentElement.firstChild as HTMLMetaElement).content as unknown as Element).querySelectorAll("script")) {
+      const activatedScriptElement = this.createScriptElement(inertScriptElement)
+      inertScriptElement.replaceWith(activatedScriptElement)
+    }
+  }
+
+  createScriptElement(element: Element) {
```

Comment:
Could `createScriptElement` from the [core/renderer.ts](https://github.com/hotwired/turbo/blob/e2d53054438fa16b4babf6a7afeba7a828764772/src/core/renderer.ts#L47) be reused ?

Reply:
Hi @jean-francois-labbe ! Thanks for the advice! I wish I knew how to do it...
Learning and writing in TS is torture for me

Reply:
should I create a new class similarly to the `FrameRenderer` ? Smth like
```
export class StreamRenderer extends Renderer<StreamElement> {
```

Reply:
aaand tests are now refusing to start
```
➜  turbo git:(main) ✗ yarn test
yarn run v1.22.17
$ NODE_OPTIONS=--inspect node src/tests/runner.js
Debugger listening on ws://127.0.0.1:9229/767ccf04-609a-4577-8e35-0e2cb7ace1b5
For help, see: https://nodejs.org/en/docs/inspector
Listening on localhost:9000 (ws 9001)
Tunnel started
BUG: suiteEnd was received for invalid session 
(ノಠ益ಠ)ノ彡┻━┻
SessionNotCreatedException: [POST http://localhost:4444/wd/hub/session / {"desiredCapabilities":{"name":"intern","idle-timeout":60,"browserName":"chrome","goog:chromeOptions":{"args":["headless","disable-gpu","no-sandbox"]},"browser":"chrome"}}] session not created: This version of ChromeDriver only supports Chrome version 99
Current browser version is 101.0.4951.54 with binary path /usr/bin/google-chrome
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'localhost', ip: '127.0.0.1', os.name: 'Linux', os.arch: 'amd64', os.version: '5.17.0-5.1-liquorix-amd64', java.version: '1.8.0_131'
Driver info: driver.version: unknown
```
Jesus Christ, what an experience...

Reply:
Oh, I'll try to help you.
I'd create a class such as `scriptActivator` or `scriptInterceptor` to give it a home.
I'd use FormSubmission as an example https://github.com/hotwired/turbo/blob/main/src/core/drive/form_submission.ts#L43
As an example.
It is used here https://github.com/hotwired/turbo/blob/e30d36bf56d020d63d87a8350d9d0361edd9b804/src/core/frames/frame_controller.ts#L173

I have no idea if that's what is expected.

Reply:
hooray, fixed the tests execution error by adding this to intern.json:
```
"tunnelOptions": {
    "drivers": [
      { "name": "chrome", "version": "101.0.4951.41" }
    ]
  }
  ```
  not sure if I should add it to the PR though

Reply:
@jean-francois-labbe, Thank you for the help, but this is still too complicated for me, taking into account my little knowledge in statically typed languages, including TypeScript, and in the Turbo ecosystem. Implementing the new `class ScriptActivator` that would use a small part of the abstract `class Renderer`  following the example of `FormSubmission` (which doesn't exactly implement any abstract classes) is greek to me.

I suggest we make compromise: what if we make the `createScriptElement` function exportable in the `renderer.ts` . And then we import that function to `stream_message.ts` so it would be reused?

Please, check my solution out, just pushed a new commit to the PR. Looking forward to your reply!